XenModel/XenAPI/FriendlyErrorNames.resx

```diff
@@ -819,6 +819,9 @@
   <data name="PIF_DOES_NOT_ALLOW_UNPLUG" xml:space="preserve">
     <value>The operation you requested cannot be performed because the specified PIF does not allow unplug.</value>
   </data>
+  <data name="PIF_HAS_FCOE_SR_IN_USE" xml:space="preserve">
+    <value>The operation you requested cannot be performed because the specified PIF has fcoe sr in use.</value>

```

Comment:
I think we should use uppercase for "FCoE SR", but as this comes from the xapi errors, it would need to be fixed there first.

Reply:
Indeed